### ISSUE TITLE
False Skipped status with failed tests

### ISSUE THREAD

**Issue by pagmoro** [prashant-ramcharan/courgette-jvm#247]
Reference: https://github.com/prashant-ramcharan/courgette-jvm/issues/245

Hi @prashant-ramcharan ,
Unfortunately this issue cropped up again in 5.9.0 too , not sure what changed. I can confirm that it was working earlier.
I was unable to reproduce in the same project you provided.

Our gradle builds are succeeding even with failing tests. For now , I have disabled courgette-jvm , but any pointers as to what might be the issue? I went back and forth between multiple versions and all have the same issue.

**Comment by prashant-ramcharan** [comment]
Hi @pagmoro 

I cannot reproduce this and the build fails when there are failing tests.

Do you have the following in your gradle task?

````gradle
outputs.upToDateWhen { false }
````

**Run using Gradle**
````
suites.junit.ScenarioSuite > courgette.api.junit.Courgette.Feature 1 FAILED
    courgette.runtime.CourgetteTestFailureException

4 tests completed, 1 failed

> Task :runScenariosUsingJUnit FAILED

Execution failed for task ':runScenariosUsingJUnit'.
````

**Comment by prashant-ramcharan** [comment]
Please provide a demo project that I can reproduce this with.

**Comment by pagmoro** [comment]
Hi @prashant-ramcharan I do not have ignoreFailures (its only set for custom plugins ) in my build gradle , neither is the outputs.upToDateWhen set anywhere. I could not reproduce it on the sample project that you provided for the earlier issue.

Unfortunately , I've had to disable courgette-jvm in our project for now , though I will re-enable and try it at a later date once we have added more Cukes. Thank you.
